src/components/assets/NativeAssetList.vue

```diff
@@ -81,7 +81,7 @@
             <div class="column--balance">
               <div v-if="!isSkeleton" class="column__box">
                 <span class="text--value">
-                  <TokenBalance :balance="numEvmDeposit" :symbol="nativeTokenSymbol" />
+                  <TokenBalance :balance="String(numEvmDeposit)" :symbol="nativeTokenSymbol" />
```

Comment:
I see that the `numEvmDeposit` type is `number`. Is this big enough to display the token amount? I maybe overthinking this (in which case, you can safely ignore my comment), but I feel like casting to a string isn't the best way.
What do you think about using `BigNumber` type for all values?

Reply:
`numEvmDeposit` is already formatted to ASTR from wei, so we don't need to consider BigNumber (I use BigNumber whenever the app has to calculate with 'wei' unit only).

Reply:
There is no case for passing the amount with wei unit to `TokenBalance` component, so I don't think we need to cast the amount to BigNumber. `TokenBalance` displays the amount of the token and unit like `100 ASTR`.